velox/vector/arrow/Bridge.cpp

```diff
@@ -1399,10 +1457,37 @@ VectorPtr createTimestampVector(
   const auto* rawNulls = nulls->as<const uint64_t>();
 
   if (length > nullCount) {
-    for (size_t i = 0; i < length; ++i) {
-      if (!bits::isBitNull(rawNulls, i)) {
-        rawTimestamps[i] = Timestamp::fromNanos(input[i]);
-      }
+    switch (options.timestampUnit) {
```

Comment:
The timestamp unit of the input timestamp Arrow vector may be inconsistent with the one in the options. In this case, we can choose to either throw an error or prioritize the schema. Prioritizing the `options.timestampUnit` might lead to incorrect data without any awareness.

Reply:
Thank you for noticing the risk here.
> We can choose to either throw an error or prioritize the schema.

I assume throwing an error could help avoid implicit unexpected behavior when mismatch occurs. Another option might be to follow schema totally when importing from Arrow. I feel this might be better as Schema already provides the unit. How do you think?

Reply:
I agree. Updated this PR to follow arrow schema when importing from arrow.